src/mdns/mdns_avahi.cpp

```diff
@@ -1170,10 +1173,15 @@ void PublisherAvahi::HostSubscription::HandleResolveResult(AvahiRecordBrowser *
     bool       resolved   = false;
     int        avahiError = AVAHI_OK;
 
+    otbrLog(aEvent != AVAHI_BROWSER_FAILURE ? OTBR_LOG_INFO : OTBR_LOG_WARNING, OTBR_LOG_TAG,
+            "resolve host reply: %s inf %d protocol %d class %" PRIu16 " type %" PRIu16 " size %zu flags %d event %d",
+            aName, aInterfaceIndex, aProtocol, aClazz, aType, aSize, static_cast<int>(aFlags),
```

Comment:
not related to this PR, however for the formal parameter name`aClazz`, is it intentionally or a typo?

Reply:
It's intentional to stay consistent with avahi's callback declaration. We can name it `aClass` here though. The reason avahi used `clazz` is that `class` cannot be a variable name.
```
typedef void (*AvahiRecordBrowserCallback) (
    AvahiRecordBrowser *b,
    AvahiIfIndex interface,
    AvahiProtocol protocol,
    AvahiBrowserEvent event,
    const char *name,
    uint16_t clazz,
    uint16_t type,
    const void *rdata,
    size_t size,
    AvahiLookupResultFlags flags,
    void *userdata);
```